askbot/templates/editors/rich_text.html

```diff
@@ -30,7 +30,7 @@
                 django.jQuery = $;
             </script>
             <script type="text/javascript">
-                askbot['settings']['tinyMCEPlugins'] = JSON.parse('{{ settings.TINYMCE_PLUGINS|as_json }}');
+                askbot['settings']['tinyMCEPlugins'] = JSON.parse('{{ TINYMCE_PLUGINS|as_json }}');
```

Comment:
I've checked code of django-tinymce - it does not provide context variable TINYMCE_PLUGINS.
Askbot is doing that in askbot/context.py - it adds "settings" context variable, which contains the TINYMCE_PLUGINS key.

It is possible to edit TINYMCE_DEFAULT_SETTINGS in the settings.py and add/remove plugins.

Thanks, I'll remove the duplicate entry, but retain the "settings" reference for now.